src/NServiceBus.AcceptanceTesting/AcceptanceTestingTransport/AcceptanceTestingTransportExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace NServiceBus
+{
+    public static class AcceptanceTestingTransportExtensions
+    {
+        public static TransportExtensions<AcceptanceTestingTransport> UseNativeDelayedDelivery(this TransportExtensions<AcceptanceTestingTransport> config, bool useNativeDelayedDelivery)
```

Comment:
=> `DisableNativeDelayedDelivery()` ?

Reply:
not sure about this. It would fit better for the default value but the current API makes more sense in both directions, e.g.
`UseNativeDelayedDelivery(true/false)` (instead of double negations like `DisableDelayedDelivery(false)`.

Reply:
Not following, just have  a `.DisableDelayedDelivery()` with no params should be enough?

Reply:
I'd prefer to have a bool parameter on that config option. Have a look at the current implementation to see how it's used.